plugins/modules/webhook.py

```diff
@@ -0,0 +1,161 @@
+    type: str
+  event:
+    description:
+      - Must be one of: actions.katello.content_view.promote_succeeded, actions.katello.content_view.publish_succeeded, actions.katello.repository.sync_succeeded, actions.remote_execution.run_host_job_ansible_configure_cloud_connector_succeeded, actions.remote_execution.run_host_job_ansible_enable_web_console_succeeded, actions.remote_execution.run_host_job_ansible_run_capsule_upgrade_succeeded, actions.remote_execution.run_host_job_ansible_run_host_succeeded, actions.remote_execution.run_host_job_ansible_run_insights_plan_succeeded, actions.remote_execution.run_host_job_ansible_run_playbook_succeeded, actions.remote_execution.run_host_job_foreman_openscap_run_oval_scans_succeeded, actions.remote_execution.run_host_job_foreman_openscap_run_scans_succeeded, actions.remote_execution.run_host_job_katello_errata_install_by_search_succeeded, actions.remote_execution.run_host_job_katello_errata_install_succeeded, actions.remote_execution.run_host_job_katello_group_install_succeeded, actions.remote_execution.run_host_job_katello_group_remove_succeeded, actions.remote_execution.run_host_job_katello_group_update_succeeded, actions.remote_execution.run_host_job_katello_host_tracer_resolve_succeeded, actions.remote_execution.run_host_job_katello_module_stream_action_succeeded, actions.remote_execution.run_host_job_katello_package_install_by_search_succeeded, actions.remote_execution.run_host_job_katello_package_install_succeeded, actions.remote_execution.run_host_job_katello_package_remove_succeeded, actions.remote_execution.run_host_job_katello_package_update_succeeded, actions.remote_execution.run_host_job_katello_packages_remove_by_search_succeeded, actions.remote_execution.run_host_job_katello_packages_update_by_search_succeeded, actions.remote_execution.run_host_job_katello_service_restart_succeeded, actions.remote_execution.run_host_job_leapp_preupgrade_succeeded, actions.remote_execution.run_host_job_leapp_remediation_plan_succeeded, actions.remote_execution.run_host_job_leapp_upgrade_succeeded, actions.remote_execution.run_host_job_puppet_run_host_succeeded, actions.remote_execution.run_host_job_rh_cloud_connector_run_playbook_succeeded, actions.remote_execution.run_host_job_rh_cloud_remediate_hosts_succeeded, actions.remote_execution.run_host_job_run_script_succeeded, actions.remote_execution.run_host_job_succeeded, actions.remote_execution.run_hosts_job_running, actions.remote_execution.run_hosts_job_succeeded, build_entered, build_exited, content_view_created, content_view_destroyed, content_view_updated, domain_created, domain_destroyed, domain_updated, host_created, host_destroyed, host_updated, hostgroup_created, hostgroup_destroyed, hostgroup_updated, model_created, model_destroyed, model_updated, status_changed, subnet_created, subnet_destroyed, subnet_updated, user_created, user_destroyed, user_updated
+      - Required when creating a new webhook
+    type: str
+  webhook_template_id:
```

Comment:
this would require the user to know the DB id to pass it.
better we make it `webhook_template` and let the API look it up?